redirects.json

```diff
@@ -103,7 +103,7 @@
   },
   {
     "source":"/ui-legacy/interactions/chatbot/<*>",
-    "target":"https://ui.docs.amplify.aws/components/chatbot",
+    "target":"https://ui.docs.amplify.aws/angular/getting-started/migration#update-and-usage-1",
```

Comment:
https://ui.docs.amplify.aws/react/components/chatbot 404s, and it looks like chatbot support was deprecated as [noted here](https://ui.docs.amplify.aws/angular/getting-started/migration#update-and-usage-1:~:text=The%20following%20deprecated%20components%20imported%20from%20%40aws%2Damplify/ui%2Dangular/legacy%20are%20removed%3A)